velox/exec/Window.cpp

```diff
@@ -185,7 +185,7 @@ void Window::addInput(RowVectorPtr input) {
   numRows_ += input->size();
 }
 
-inline bool Window::compareRowsWithKeys(
+bool Window::compareRowsWithKeys(
```

Comment:
Why do we need this change?

Reply:
It seems the subclass cannot call an inline method in a parent class. I encounter the following compile issue:
```
In file included from ../.././velox/exec/StreamingWindow.h:20,
                 from ../../velox/exec/StreamingWindow.cpp:16:
../.././velox/exec/Window.h:127:15: error: inline function ‘bool facebook::velox::exec::Window::compareRowsWithKeys(const char*, const char*, const std::vector<std::pair<unsigned int, facebook::velox::core::SortOrder> >&)’ used but never defined [-Werror]
  127 |   inline bool compareRowsWithKeys(
      |               ^~~~~~~~~~~~~~~~~~~
cc1plus: error: unrecognized command line option ‘-Wno-stringop-overread’ [-Werror]

```